### PR TITLE
Rust Hack and Learn blog post

### PR DESCRIPTION
**Pull request by skade**
Draft blogpost for the Rust Hack and Learn joining OTS.

Many of the authors pictures link to the opentechschool team page, should I also put an entry there?

### CODE REVIEW

File: _posts/2015-12-10-rust-hack-and-learn-joins-ots.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+authors:
+  - florian
+image: /assets/content/rust-logo-256x256-blk.png
+---
+
+Our regular [Rust](http://rust-lang.org) Hack and Learn meetup has now [officially joined](http://www.meetup.com/opentechschool-berlin/events/227320730/) the OpenTechSchool umbrella. We meet biweekly and learn the new programming language created by Mozilla. The Rust Hack and Learn is coached by Florian Gilcher, member of the global Rust community team, Johan Hoffman, founder of the group and Florian Hartwig.
+
+We make this move because we wanted to join an organisation that supports us in our goals and provides a helpful framework. OpenTechSchool runs many events with a similar concept and promotes them, which allows us to be more effective in our outreach by doing it together with other language communities.
+
+Finally, many of us come from other language communities and see a lot of value in collaborating instead of telling juvenile jokes about each other.
```
Comment (lehmannro) [inline _posts/2015-12-10-rust-hack-and-learn-joins-ots.md]: I know this is intended to be tongue-in-cheek, but written content doesn't transport that especially well.  Maybe hyperbole would work (s/telling juvenile jokes/fighting/), maybe just leave it off and emphasize collaboration (+= "learning from each other.")

File: _posts/2015-12-10-rust-hack-and-learn-joins-ots.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - global
+  - rust
+authors:
+  - florian
+image: /assets/content/rust-logo-256x256-blk.png
+---
+
+Our regular [Rust](http://rust-lang.org) Hack and Learn meetup has now [officially joined](http://www.meetup.com/opentechschool-berlin/events/227320730/) the OpenTechSchool umbrella. We meet biweekly and learn the new programming language created by Mozilla. The Rust Hack and Learn is coached by Florian Gilcher, member of the global Rust community team, Johan Hoffman, founder of the group and Florian Hartwig.
+
+We make this move because we wanted to join an organisation that supports us in our goals and provides a helpful framework. OpenTechSchool runs many events with a similar concept and promotes them, which allows us to be more effective in our outreach by doing it together with other language communities.
```
Comment (lehmannro) [inline _posts/2015-12-10-rust-hack-and-learn-joins-ots.md]: "... which ... by ..." longcat.  Split up?

File: _config.yml [hunk 3]
```diff
@@ -63,6 +63,12 @@ authors:
     email: duana at OpenTechSchool dot org
     image: http://www.opentechschool.org/images/team/duana.jpg
     about: is Software Developer at Soundcloud and initiator of the Learners-meetup concept. As such she is organising this and other OpenTechSchool events.
+  florian:
+    name: Florian Gilcher
+    gplus: 103019814974030696862
+    email: florian at OpenTechSchool dot org
+    image: http://www.opentechschool.org/images/team/florian.jpg
```
Comment (lehmannro) [inline _config.yml:70]: Yup, this probably needs a PR to our main [OTS site repository](http://github.com/OpenTechSchool/www.opentechschool.org).

**Comment by lehmannro** [comment]
Proofread.  Is there any particular date you want to get this live, or ASAP?

File: _posts/2015-12-10-rust-hack-and-learn-joins-ots.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
+---
+published: true
+layout: post
+title: "Rust Hack and Learn joins OTS"
+tags:
+  - global
```
Comment (lehmannro) [inline _posts/2015-12-10-rust-hack-and-learn-joins-ots.md]: Isn't it more `berlin` than `global`?

**Comment by skade** [comment]
I think @xMartin would be happy if it were live soon. I'll also post the pull request to the OTS page.

**Comment by skade** [comment]
There we are: https://github.com/OpenTechSchool/www.opentechschool.org/pull/209

File: _posts/2016-01-15-rust-hack-and-learn-joins-ots.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
+---
+published: true
+layout: post
+title: "Rust Hack and Learn joins OTS"
+tags:
+  - global
+  - rust
```
Comment (xMartin) [inline _posts/2016-01-15-rust-hack-and-learn-joins-ots.md:7]: should we add "berlin"?

**Comment by xMartin** [comment]
@skade thank you! I think it would be helpful to point out that this is happening in berlin as the blog has a global audience. while it is relevant for global ots that there is now a rust thing in our universe it is a bit hard to find out where this is currently actually happening.

File: _posts/2016-01-15-rust-hack-and-learn-joins-ots.md (hunk 5 above)
Comment (lehmannro) [inline _posts/2016-01-15-rust-hack-and-learn-joins-ots.md:7]: …and remove "global".

**Comment by skade** [comment]
I changed the tags and clarified the text.

**Comment by xMartin** [comment]
Thanks you and welcome to the family, again :)

Will put you on the Berlin team page as well.

**Pull request merged**